guides/common/modules/con_client-operating-systems.adoc

```diff
@@ -12,21 +15,11 @@ For example:
 * Operating system installers that can perform unattended installations, such as Anaconda or Debian-installer
 * Other clients where integration is provided by external plugins
```

Comment:
I don't understand what this is about. Other OSs? Other client features? (non-blocking for this PR)

Reply:
I didn't understand this either; I had trouble to come up with better wording, so I left things mostly unchanged. I am open to dropping line 15+16. What do you think?

Reply:
Probably a good idea.

Reply:
Applied.